plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/BaseMySqlConnectorTest.java

```diff
@@ -196,6 +197,66 @@ public void testShowCreateTable()
                         ")");
     }
 
+    @Test
+    public void testCreateTableWithPrimaryKey()
+    {
+        String catalogName = getSession().getCatalog().orElseThrow();
+        String schemaName = getSession().getSchema().orElseThrow();
+        String tableName = "test_create_with_primary_key_" + randomNameSuffix();
+        // create table with a primary key
+        assertUpdate("CREATE TABLE " + tableName + " (a bigint NOT NULL, b bigint, c bigint) with (primary_key = ARRAY['a'])");
+        assertThat(computeScalar(format("SHOW CREATE TABLE %s", tableName)))
+                // If the connector reports additional column properties, the expected value needs to be adjusted in the test subclass
+                .isEqualTo(format(
+                        """
+                        CREATE TABLE %s.%s.%s (
+                           a bigint NOT NULL,
+                           b bigint,
+                           c bigint
+                        )
+                        WITH (
+                           primary_key = ARRAY['a']
+                        )\
+                        """,
+                        catalogName,
+                        schemaName,
+                        tableName));
+        assertUpdate("DROP TABLE " + tableName);
+        // create table with multiple primary keys
+        assertUpdate("CREATE TABLE " + tableName + " (a bigint NOT NULL, b bigint NOT NULL, c bigint) with (primary_key = ARRAY['a', 'b'])");
+        assertThat(computeScalar(format("SHOW CREATE TABLE %s", tableName)))
```

Comment:
We can use + operator in this case: https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#prefer-string-formatting